### DISCUSSION TITLE
Clone feature on LED display version.

### DISCUSSION THREAD

**Discussion by sckramer** [keirf/flashfloppy#705] [General]
Any chance clone (simplified) could be enabled on the LED version... maybe while it's in eject mode or some other creative way to get it in with only 3 digits and 2 buttons?

**Comment by keirf** [comment]
It would be hard to work out a UI for it that makes sense.

**Comment by sckramer** [comment]
How about:

twobutton-action = zero* | eject | htu | rotary | rotary-fast | reverse | **clone**

clone: Clone image in Slot 0*, added as ZZZ_Clone000.img**

*Slot 0 has a blank 1.44 etc (anything)
**Filename to make it the last image in the file list... or require indexed mode?

Basically a limited clone mode, on rails. But you can fire off a new image on the spot... then clean up / manage it later externally.

Thanks!
